doc/services/smf/index.rst

```diff
@@ -13,6 +13,25 @@ provides an easy way for developers to integrate state machines into their
 application. The framework can be added to any project by enabling the
 :kconfig:option:`CONFIG_SMF` option.
 
+UML State Machines
+==================
+
+SMF will follow UML hierarchical state machine rules for transitions only when
```

Comment:
Is there a reference UML specification that you could link to?

Reply:
Done.